packages/evm/src/opcodes/util.ts

```diff
@@ -274,3 +279,280 @@ export function exponentiation(bas: bigint, exp: bigint) {
   }
   return t
 }
+
+const SUB_MEM_USAGE_COST_PRECALC_CONSTS = [
+  BigInt(0),
```

Comment:
I am not really a fan of this. If one of these constants are wrong, then we have to be "lucky" that the test suite catches this. Can these constants be initialized by calling into `subMemUsage`? (Probably need to just extract the formula `cost = words * fee + (words * words) / quadCoeff` and call into this with the right constants - note: they have not been changed by any hardfork so far and these are `3` for `fee` and 512 for `quadCoeff`).

Reply:
I have created the list programmatically by running the formula and programmatically creating a string of the array, so risk should be alread pretty low, I would not know what should have gone wrong there.

What makes this a non issue though is that - on real world blocks - this is simply iterating from 0 to 273 and then starting all over again endlessly. 😋 (totally didn't know about this behavior)

So this is fully tested by a small mainnet run with contract execution.